src/Controls/src/Core/Button/Button.iOS.cs

```diff
@@ -33,6 +33,10 @@ private static void MapPadding(IButtonHandler handler, Button button)
 		public static void MapText(IButtonHandler handler, Button button)
 		{
 			handler.PlatformView?.UpdateText(button);
+
+			// Update all of the attributed text formatting properties
```

Comment:
Firsr of all, thanks for the contributions.
In Controls and in Core, we are mapping the Text to a extension method. In Core, invoke the `MapFormatting `method that update the `AttributedTitle `taking into account the character spacing and the text color. Instead create a new method with a similar logic, everything should be in the same existing method un Core ButtonExtensions, here: https://github.com/dotnet/maui/blob/977642b689f1af22afe0f4cbaf21b6d8106f648c/src/Core/src/Platform/iOS/ButtonExtensions.cs#L51
This also will avoid the publicAPI small breaking changes.

I think, just need to set the font family and size there. The rest of changes, the test etc is fine.

Reply:
Thank you for this advice! I've refactored the code and hope it is what you want